src/handler/s3Event.ts

```diff
@@ -30,13 +33,14 @@ const handleEvent = async (record: S3EventRecord) => {
 export const handler = async (
   event: S3Event,
 ): Promise<Record<string, unknown>> => {
-
   for (const record of event.Records) {
     try {
       await handleEvent(record);
-    } catch  (err) {
+    } catch (err) {
       logger.error(err);
-      logger.error(`The file ${record.s3.object.key} failed somewhere, 500 was returned.`);
```

Comment:
Out of curiosity, is it your local linter that wraps these lines?

There is a lot of refactoring which makes it hard to follow the actual code that was changed for the purpose of the PR/ticket. Also, this line is not even 100 characters long 😃

Reply:
Fair enough, thanks for the info.